src/pager/pager.spec.ts

```diff
@@ -131,6 +145,23 @@ describe('ngb-pagination', () => {
            expectPager(fixture.nativeElement, [true, true]);
          });
        })));
+
+    it('should have pager-prev and pager-next classes set', async(inject([TestComponentBuilder], (tcb) => {
+         const html = '<ngb-pager [noOfPages]="noOfPages" page="5" [alignLinks]=true></ngb-pager>';
+
+         tcb.overrideTemplate(TestComponent, html).createAsync(TestComponent).then((fixture) => {
+           fixture.detectChanges();
+           expectPager(fixture.nativeElement, [true, true]);
+           expectAlignClasses(fixture.nativeElement, [true, true]);
+
+           fixture.componentInstance.noOfPages = 5;
+           fixture.componentInstance.page = 5;
+           fixture.detectChanges();
+           expectPager(fixture.nativeElement, [true, false]);
```

Comment:
This section of the above 4 lines is not needed, as already covered in other tests. What we should have instead is a change in the `alignLinks` binding to see that CSS classes are added / removed.